src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const styleObject = {};
+  const styleCommands = sourceString.split(';');
+
+  styleCommands.forEach((element) => {
+    const [key, value] = element.split(':');
+
+    if (key && value) {
+      styleObject[key.trim()] = value.trim();
+    }
+  });
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes code much more readable. Consider refactoring the code to chain the `split` and `forEach` methods with proper indentation.